source/administration/command-line-tools.rst

```diff
@@ -191,7 +209,7 @@ platform channel restore
 ~~~~~~~~~~~~~~~~~~~~~~~~
 
   Description
-    Restore a previously deleted channel. Channels can be specified by {team}:{channel} using the team and channel names or IDs.
+    Restore a channel from the archive. Channels can be specified by {team}:{channel} using the team and channel names, or by using channel IDs.
```

Comment:
Maybe we should say urls or handles instead of names? Whatever is more consistent with the rest of the docs and product. 

Can we also add this sentence to the description of the channel archive command?